Source/AssetRipper.SourceGenerated.Extensions/AnimationCurveExtensions.cs

```diff
@@ -76,11 +76,15 @@ public static void SetValues(this IAnimationCurve_Single animationCurve, UnityVe
 			}
 		}
 
-		public static void SetDefaultRotationOrderAndCurveLoopType(this IAnimationCurve_Vector3f animationCurve)
+		public static void SetDefaultCurveLoopType(this IAnimationCurve_Vector3f animationCurve)
 		{
 			animationCurve.PreInfinityE = CurveLoopTypes.CycleWithOffset;
 			animationCurve.PostInfinityE = CurveLoopTypes.CycleWithOffset;
-			animationCurve.RotationOrderE = RotationOrder.OrderZXY;
+		}
+
+		public static void SetRotationOrder(this IAnimationCurve_Vector3f animationCurve, RotationOrder rotationOrder = RotationOrder.OrderZXY)
+		{
+			animationCurve.RotationOrderE = rotationOrder;
 		}
```

Comment:
Change this back please, and just use `animation.Curve.RotationOrderE = ` elsewhere. There's no harm in setting the value twice.